src/app/index.php

```diff
@@ -62,7 +62,7 @@ function (Request $request, Response $response, array $args) {
 <div class="container">
   <h3>Welcome to HypeJab! 💉 😃 </h3>
   <p>HypeJab is a deliberately vulnerable web application intended for benchmarking automated scanners.</p>
-  <p>v0.0.3</p>
+  <p>v0.0.5</p>
```

Comment:
@karthikuj  please add an entry in changeling, like the changes are currently under `Unreleased` section, please create an entry for 0.0.5

Reply:
Done.